docs/modules/ROOT/pages/openshift-service-mesh/tutorial-bookinfo.adoc

```diff
@@ -0,0 +1,220 @@
+= Getting started with OpenShift Service Mesh
+
+This tutorial explains how to use the OpenShift Service Mesh to manage the bookinfo demo application.
+
+
+== Requirements
+
+To follow this guide, please make sure that you have the following tools installed:
+
+`oc`:: You can download the OpenShift command directly from OpenShift Web Console. Open the help menu (marked as a question mark at the top right) and select the "Command line tools" entry.
+
+
+== Step 1: Creating the ServiceMeshControlPlane (requires cluster-admin)
```

Comment:
Afaik this doesn't require cluster-admin with OpenShift Service Mesh. Project admin should grant permissions to create ServiceMeshControlPlane and ServiceMeshMemberRoll objects.

Reply:
I needed it on the lab. But that's fair.

Reply:
From what I can see, it shouldn't be necessary on the lab either. It's possible that you weren't fully part of the organization yet when you tested it.